src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
```

Comment:
I merged Snehasish branch with mine and then committed changes. So some files maybe same for both of us